ironfish/src/blockchain/nullifierSet/nullifierSet.test.ts

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Assert } from '../../assert'
+import {
+  createNodeTest,
+  useAccountFixture,
+  useBlockWithTx,
+  useMinerBlockFixture,
+} from '../../testUtilities'
+import { NullifierSet } from './nullifierSet'
+
+describe('NullifierSet', () => {
```

Comment:
Should we just move these all into blockchain.test.ts, since they are blockchain tests?